run.sh

```diff
@@ -116,8 +119,12 @@ main() {
 
   h3 "Adjusting file permissions"
   groupadd -f docker && usermod -aG docker www-data
-  find /app -type d -exec chmod 755 {} \;
-  find /app -type f -exec chmod 644 {} \;
+  if [[ "${PHP_ENV}" =~ ^dev ]]; then
+    chmod -R 777 /app
```

Comment:
I think that having different permissions in development is a bad idea, it will cause issue when you deploy. whyt @dsifford ?